lib/quickbooks/service/base_service.rb

```diff
@@ -264,6 +258,23 @@ def do_http(method, url, body, headers) # throws IntuitRequestException
         end
 
         response = Quickbooks::Service::Responses::OAuthHttpResponse.wrap(raw_response)
+
+        log(
+          <<~QBO_LOGGER
+            ------------------ REQUEST --------------------
+            METHOD: #{method}
+            HEADERS: #{headers.inspect}
+            BODY:
+            #{request_body(body)}
+            ------------------ RESPONSE -------------------
+            RESPONSE CODE: #{response.code}
+            HEADERS: #{response.respond_to?(:headers) ? response.headers : ''}
+            BODY:
+            #{response_body(response)}
+            -----------------------------------------------
+          QBO_LOGGER
+        )
```

Comment:
https://github.com/ruckus/quickbooks-ruby/pull/490/files
This got pulled into the gem after we forked, looks similar. If this is good then i would not make changes to this file and instead update this fork from latest tag. Something like this:https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/syncing-a-fork

Reply:
Looking further, the tax issue that we fixed on this fork is fixed as well
https://github.com/ruckus/quickbooks-ruby/pull/495/files
So i guess, get rid of this fork and update the original gem to latest.